plugins/source/azure/client/spec/spec.go

```diff
@@ -12,15 +12,15 @@ import (
 type Spec struct {
 	// Specify which subscriptions to sync data from.
 	// Empty means all visible subscriptions.
-	Subscriptions []string `json:"subscriptions" jsonschema:"minLength=1,uniqueItems=true"`
+	Subscriptions []string `json:"subscriptions" jsonschema:"minLength=1,uniqueItems=true,example=subscription_id"`
```

Comment:
Shouldn't the examples be real examples, like `00000000-0000-0000-0000-000000000000`?